src/components/CMessageRow.vue

```diff
@@ -82,21 +85,26 @@ export default {
       if (!this.startedDatetime || !this.enqueuedDatetime) {
         return null;
       }
-      return formatDistance(this.enqueuedDatetime, this.startedDatetime);
+      const diff = utils.getDistance(this.startedDatetime, this.enqueuedDatetime);
+      return `${diff.hours}:${diff.minutes}:${diff.seconds}`;
+    },
+    remainingTime() {
+      if (this.endDatetime || !this.startedDatetime) {
```

Comment:
could you display remainingTime and progress only if state=Started, else it does not make sense